W3ChampionsStatisticService/Ladder/PlayerInfoForGlobalSearch.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Serialization;
+using W3C.Domain.CommonValueObjects;
+using W3ChampionsStatisticService.Ladder;
+
+namespace W3ChampionsStatisticService.Ladder
+{
+    public class PlayerInfoForGlobalSearch
+    {
+        [JsonIgnore]
+        public RankInfo RankInfo { get; set; }
```

Comment:
would it be better to have all the `RankInfo` for a player at a list here, rather than creating multiple `PlayerInfoForGlobalSearch` objects in a list?

i.e. 

returning a player with a list of all their rank infos over multiple seasons and gamemodes
vs
returning multiple objects that represent a players ranking all in different seasons